doc/build-osx.md

```diff
@@ -86,26 +94,34 @@ Mining is also possible in disable-wallet mode using the `getblocktemplate` RPC
 Bitcoin Core is now available at `./src/bitcoind`
 
 Before running, you may create an empty configuration file:
+
 ```shell
 mkdir -p "/Users/${USER}/Library/Application Support/Bitcoin"
-
+```
+```shell
 touch "/Users/${USER}/Library/Application Support/Bitcoin/bitcoin.conf"
-
+```
+```shell
```

Comment:
why are these split?

Reply:
The underlying issue is related to: https://help.github.com/en/articles/creating-and-highlighting-code-blocks#fenced-code-blocks

The syntax highlighting isn't standard markdown.
This is creating an inconsistency with markdown interpreters. The GitHub interpreter seems to correct this (probably because the error is very common). 

I will be happy to correct them because it isn't isolated to this one file.

![Screen Shot 2019-10-02 at 12 51 48 PM](https://user-images.githubusercontent.com/152159/66064790-18d17e80-e514-11e9-8cde-a7b27b8a11bf.png)
![Screen Shot 2019-10-02 at 12 52 23 PM](https://user-images.githubusercontent.com/152159/66064791-18d17e80-e514-11e9-9816-a0ab0a5a74f2.png)
![Screen Shot 2019-10-02 at 12 53 22 PM](https://user-images.githubusercontent.com/152159/66064793-18d17e80-e514-11e9-8a7a-80564bf8a092.png)